composer.json

```diff
@@ -61,8 +61,25 @@
     }
   },
   "scripts": {
+    "analytics": "vendor/bin/psalm",
     "check": "@cs",
+    "coverage": "@test --coverage-html coverage/",
     "cs": "php-cs-fixer fix -v --diff --dry-run",
-    "cs-fix": "php-cs-fixer fix -v --diff"
+    "cs-fix": "php-cs-fixer fix -v --diff",
+    "pr": [
```

Comment:
I think, this should be `check` instead of `pr` and the existing `check` can be removed (it's only an alias anymore)

Reply:
Done